ARGO/include/GameScreen.h

```diff
@@ -44,7 +45,8 @@ class GameScreen
 	Controller m_controllers[Utilities::S_MAX_PLAYERS];
 	Entity m_players[Utilities::S_MAX_PLAYERS];
 	std::vector<Entity> m_entities;
-	std::vector<Entity> m_levelTiles;
+	Entity m_goal;
+	SDL_Renderer* m_renderer;
```

Comment:
you don't need this, the create player function already passes in the renderer

Reply:
I needed it for the create enemy function

Reply:
Github, shrunk the functions so I was seeing the create enemy function at the end of create player function, however you should probably change create enemy function to take in the renderer cause its being passed into the init/reset functions anyway

Reply:
I'll be changing this around when I'm changing the enemies into a pool so I'll redo this then since I'll probably move it all into a manager, so it's fine for now.